polygeist/tools/cgeist/Lib/TypeUtils.cc

```diff
@@ -99,6 +99,27 @@ mlir::Type getPtrTyWithNewType(mlir::Type Orig, mlir::Type NewElementType) {
       .Default([](auto) -> mlir::Type { llvm_unreachable("Invalid type"); });
 }
 
+mlir::sycl::AccessAddrSpace getAccAddressSpace(const int AddrSpace) {
+  switch (AddrSpace) {
```

Comment:
This function can turn into `static_cast<mlir::sycl::AccessAddrSpace>(AddSpace)`?

Reply:
Yes, that can happen